File: zerojudge/b969-hello-everyone/solution.py
"""
Author: 1chooo<hugo970217@gmail.com>
Problem link: https://zerojudge.tw/ShowProblem?problemid=b969
Status: AC
"""

if __name__ == "__main__":
    names_input = str(input())
    greeting_message = str(input())

    names_list = names_input.split(" ")
    names_count = len(names_list)

    for i in range(names_count):
        print(f"{greeting_message}, {names_list[i]}")

File: zerojudge/b969-hello-everyone/solution.cpp
/*
 * Author: 1chooo<hugo970217@gmail.com>
 * Problem link: https://zerojudge.tw/ShowProblem?problemid=b969
 * Status: AC
 */

#include <cstdlib>
#include <iostream>
#include <sstream>

#define INT_MAX 2147483647

using namespace std;

int main(void) {
    string names;
    string greetings;
    string name;

    getline(cin, names);
    getline(cin, greetings);

    stringstream splitNames(names);

    while (splitNames >> name) {
        cout << greetings << ", " << name << endl;
    }

    return 0;
}
